contracts/README.md

```diff
@@ -0,0 +1,111 @@
+# Nori Smart Contracts
+
+Detailed documentation on these contracts is available [here](../docs)
+
+## ERC20 Token
+
+### NORI (NORI)
+
+The [$NORI](./NORI.sol) token is Nori's fungible token which functions as a gift card to purchase NRTs. One NRT is worth one $NORI.
+
+- Deployed on: Ethereum mainnet.
+- Initial supply minted at deployment: 500M
+- Minting and burning are disabled.
+
+[NORI on Etherscan](https://etherscan.io/token/0x961760ad1bed52bf4d79aa4b1558e7f9d72071e4) Currently live deployment: (will get replaced with a new version of the contract having ERC777 support removed.)
+
+### BridgedPolygonNORI (NORI)
+
+[BridgedPolygonNORI](./BridgedPolygonNORI.sol) is the [$NORI](./NORI.sol) token bridged to Polygon PoS.
+
+- Polygon child chain mapper is the only contract with mint / burn permission.
+- Initially deployed with totalSupply of zero.
+
+[NORI on PolygonScan](https://polygonscan.com/token/0x8cf6e82919f69ae382def8f94e581a43ce1e70c1) Currently live deployment (will get replaced with a new version of the contract having ERC777 support removed.)
+
+## Nori Marketplace
+
+### Removal (NRT)
+
+[source](./Removal.sol)
+
+ERC1155 token representing the NRTs (carbon removal credits) issued to a supplier.
+
+Each token ID encodes the wallet address of the supplier to whom the NRTs were issued in addition to information about the source and location of the carbon removed. See [RemovalLib](./RemovalIdLib.sol) for details.
+
+The total supply of each Removal token ID represent the estimated quantity of NRTs (tonnes of CO2 removed from the atmosphere) on a specific parcel of land in a specific year.
+
+Lifecycle of a Removal:
+
+1. Minted to a supplier's address
+2. Ownership transfered to the _Market_ contract to be listed for sale.
+3. Sold by the _Market_ and ownership transferred to one or more _Certificate_
```

Comment:
technically the ownership of the removal is transferred to the Certificate contract in all cases.  the accounting about which certificate IDs "own" the removal exists only in the certificate contract itself. not sure if that's worth being more specific about here

Reply:
Good point, I was simplifying that but it makes it unclear.